Feat: implemented human evaluations

ashrafchowdury opened the pull request:
### Descriptions

Implemented human evaluations on evaluation page reusing the `SingleModelEvalOverview.tsx` and `AbTestingEvalOverview.tsx` component from overview page by conditionally made them adjustable, so that both use in overview and evaluations page.

bekossy reviewed (changes requested):
Thanks for the work on this PR. Here are some changes to address:

1. Let's make it clear that both the `SingleModelEvaluation` and `AbTestingEvaluation` are reusable components and import that where needed.
2. The "Compare" button in AB Testing should navigate to the evaluation compare view.
3. We currently don’t have new changes in the Automatic Evaluation view?
4. Please resolve the merge conflicts.

ashrafchowdury commented:
**Changes summary:**
- Made it clear that both the SingleModel and AbTesting are reusable components.
- Removed the compare button from SingleModel and AbTesting after talking with Ahmed
- Some UI changes were made in the 'Create Human Eval Model'
- Merge conflict resolved

bekossy reviewed (changes requested):
Let’s also remove the previous Annotation code
`http://localhost/apps/APP_ID/annotations/single_model_test` should not be a valid path

ashrafchowdury commented:
Changes summary:
- Removed `single_model_test.tsx` and `human_a_b_testing.tsx` pages.
- Removed `AutomaticResultsEvaluation.tsx` and `HumanEvaluationResults.tsx` components.
- Adjusted cypress test according to new changes.
- Minor UI update to the 'create new human eval model'.

bekossy commented on agenta-web/cypress/e2e/app-navigation.cy.ts:
```diff
@@ -28,28 +28,10 @@ describe("App Navigation without errors", () => {
+    it("should navigate successfully to Evaluations page", () => {
+        cy.clickLinkAndWait('[data-cy="app-evaluations-link"]')
+        cy.url().should("include", "/evaluations")
+        cy.contains(/evaluations/i)
```
In addition to the app navigation test for evaluation, let's switch between all evaluation tabs and check that the URL is correct

ashrafchowdury replied:
ok
